tests/interactive/test_app.py

```diff
@@ -257,6 +257,7 @@ def callback(x: str):
         with ImplicitBuilder(expected_module.body):
             function = func.FuncOp("hello", ((index,), (index,)))
             with ImplicitBuilder(function.body) as (n,):
+                n.name_hint = "n"
```

Comment:
This will otherwise show up as a difference in `get_available_passes` below.

Reply:
Looks like this fixes a kind of bug in the test, which is that the name hints are not the same in the text and this IR